packages/kotti-ui/source/kotti-popover/KtPopoverIconTextItem.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div
+		v-bind="$attrs"
```

Comment:
Is this actually necessary? If yes, why? This should be the default behavior

Reply:
no not necessary